src/mantine-core/src/components/Accordion/AccordionItem/AccordionItem.styles.ts

```diff
@@ -1,4 +1,5 @@
 import { createStyles } from '@mantine/styles';
+import { applyHoverStyleForTouch } from '../../../utils';
```

Comment:
I think this function should be a part of `theme.fns`, usage example:

```tsx
...theme.fns.hover({ backgroundColor: 'red' })
```

(No additional documentation required)

This way function will be accessible in other packages as well.